readthedocs/search/tasks.py

```diff
@@ -210,5 +210,3 @@ def record_search_query(project_slug, version_slug, query, total_results, time_s
         version=version,
         query=query,
     )
```

Comment:
We don't need the `obj` anymore.

Reply:
but `SearchQuery.objects.create` this thing is needed right?
okay let me update the PR.

Reply:
> but SearchQuery.objects.create this thing is needed right?

yes